### PR TITLE
create-heading

### PR DESCRIPTION
**Pull request by Kushan1994**

### CODE REVIEW

File: resources/js/Components/ProjectFrontend/Completed/Complete.vue [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<script setup>
```
Comment (sureshhemal) [inline resources/js/Components/ProjectFrontend/Completed/Complete.vue:1]: File name not clear

File: resources/js/Components/ProjectFrontend/Completed/ContentFrontend.vue [hunk 2]
```diff
@@ -0,0 +1,80 @@
+<script setup></script>
```
Comment (sureshhemal) [inline resources/js/Components/ProjectFrontend/Completed/ContentFrontend.vue:1]: file name not clear

File: resources/js/Pages/Frontend/Home.vue [hunk 3]
```diff
@@ -1,7 +1,9 @@
 <script setup>
 import Carousel from '@/Components/FrontendCarousel'
+import Complete from "@/Components/ProjectFrontend/Completed/Complete";
```
Comment (sureshhemal) [inline resources/js/Pages/Frontend/Home.vue:3]: filename not clear

File: resources/js/Pages/Frontend/Home.vue [hunk 4]
```diff
@@ -1,7 +1,9 @@
...
 </script>
 
 <template>
-<Carousel></Carousel>
+    <Carousel></Carousel>
+    <Complete></Complete>
```
Comment (sureshhemal) [inline resources/js/Pages/Frontend/Home.vue:8]: <Carousel />
<Complete />

**Review by sureshhemal — CHANGES_REQUESTED** [review]

File: resources/js/Components/ProjectFrontend/Completed/Complete.vue (hunk 1 above)
Comment (Kushan1994) [reply resources/js/Components/ProjectFrontend/Completed/Complete.vue:1]: it is complete project components.  "Components/ProjectFrontend/Completed/Project.vue" That name is OK ?

Comment (sureshhemal) [reply resources/js/Components/ProjectFrontend/Completed/Complete.vue]: Lets make `Components/Projects/Completed.vue`

File: resources/js/Pages/Frontend/Home.vue (hunk 4 above)
Comment (sureshhemal) [reply resources/js/Pages/Frontend/Home.vue]: @Kushan1994 change those tags to `<Carousel />` and `<Complete />`

Comment (sureshhemal) [reply resources/js/Pages/Frontend/Home.vue]: And what is the `<Complete />` ? It doesn't make sence

File: resources/js/Components/Projects/Complete.vue [hunk 5]
```diff
@@ -0,0 +1,29 @@
+<script setup>
```
Comment (sureshhemal) [inline resources/js/Components/Projects/Complete.vue:1]: rename file as `Completed.vue`

File: resources/js/Pages/Frontend/Home.vue [hunk 6]
```diff
@@ -1,7 +1,9 @@
 <script setup>
 import Carousel from '@/Components/FrontendCarousel'
+import CompleteProject from "@/Components/Projects/Complete";
```
Comment (sureshhemal) [inline resources/js/Pages/Frontend/Home.vue:3]: completed projects

File: resources/js/Pages/Frontend/Home.vue [hunk 7]
```diff
@@ -1,7 +1,9 @@
...
 </script>
 
 <template>
-<Carousel></Carousel>
+    <Carousel />
+    <CompleteProject />
```
Comment (sureshhemal) [inline resources/js/Pages/Frontend/Home.vue:8]: completed porjects.

File: resources/js/Components/Projects/CompleteItem.vue [hunk 8]
```diff
@@ -0,0 +1,80 @@
+<script setup></script>
```
Comment (sureshhemal) [inline resources/js/Components/Projects/CompleteItem.vue:1]: file path is not correct. There is only `project` component inside `Component` folder. This file system hard to understand. Lets move all homepage components to single folder called `Components/Homepage`

So this must be `Components/Homepage/Projects/Completed.vue`

**Review by sureshhemal — APPROVED** [review]

**Review by sureshhemal — APPROVED** [review]

**Review by sureshhemal — APPROVED** [review]

**Pull request merged**